docs/cow-protocol/reference/core/auctions/accounting.md

```
@@ -35,6 +35,10 @@ Certain orders come with a list of so-called fee policies; these policies provid
 
 Protocol and partner fees are naturally denominated in the surplus token of an order, and the accounting process uses the native prices provided in each auction to convert these fee amounts to the native token of the chain; this implies that the exchange rate of these fees with respect to the native token of the chain is determined at auction creation time.
 
+#### Auction price corrections
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)